client/my-sites/people/people-section-nav/index.jsx

```diff
@@ -61,7 +61,7 @@ class PeopleSectionNav extends Component {
 				id: 'followers',
 			},
 			{
-				title: translate( 'Email Followers', { context: 'Filter label for people list' } ),
+				title: translate( 'Subscribers', { context: 'Filter label for people list' } ),
```

Comment:
I think we might need to keep the Email suffix here for a bit — "Email Subscribers"

Reply:
Done ✔️